tests/algorithms/column_pooler_test.py

```diff
@@ -101,7 +101,7 @@ def testLearnSinglePattern(self):
     self.init()
 
     object = self.generateObject(1)
-    self.learn(object, numRepetitions=1, newObject=True)
+    self.learn(object, numRepetitions=2, newObject=True)
```

Comment:
The parameters are:

~~~
      "connectedPermanenceProximal": 0.6,
      "initialProximalPermanence": 0.6,
      "synPermProximalInc": 0.1,
      "synPermProximalDec": 0.001,
~~~

The problem was that a single decrement would put it below the connected permanence, so learning two patterns would cause problems. Another fix would be

~~~python
    self.init(overrides={"synPermProximalDec": 0.0})
~~~